tests/compose-tests/test-basic-unified.sh

```diff
@@ -25,7 +25,9 @@ cat > metadata.json <<EOF
   "overrideme": "new val"
 }
 EOF
-runcompose --ex-unified-core --add-metadata-from-json metadata.json
+# Test --parent at the same time (hash is `echo | sha256sum`)
```

Comment:
It doesn't refer to a real commit object?  Won't that fail the `rev_parse`?

Reply:
`rev_parse` just needs the commit to exist if it needs to actually resolve it. If it's just a SHA256, it'll just pass through. So it already has the semantics we want.

The idea being that in the FCOS pipeline, we don't really *need* the commit object of the parent we want. We just care about its checksum.

Reply:
OK right, I forgot `ostree_repo_resolve_rev()` just passes through checksums.